src/Screens/Gameplay/Components/Energy/EnergyTab.jsx

```diff
@@ -36,9 +35,9 @@ export default function EnergyTab() {
 				{texts.gameplay.energy.energyTab.perHour}
 			</div>
 
-			<GeneralButton id='getMoreEnergyButton' onButtonClick={() => getMoreButtonClickHandler()}>
+			<div className='generalButton' id='getMoreEnergyButton' onClick={() => getMoreButtonClickHandler()}>
```

Comment:
this button wasn't uppercased, please keep it as it were.
can be done by adding `text-transform: none; border: 0px;` to it's id.